diff --git a/node_modules/babel-plugin-annotate-pure-calls/lib/index.js b/node_modules/babel-plugin-annotate-pure-calls/lib/index.js
index 2182884..0bac0b5 100644
--- a/node_modules/babel-plugin-annotate-pure-calls/lib/index.js
+++ b/node_modules/babel-plugin-annotate-pure-calls/lib/index.js
@@ -78,7 +78,7 @@ const isInAssignmentContext = path => {
 
     parentPath = _ref.parentPath;
 
-    if (parentPath.isVariableDeclaration() || parentPath.isAssignmentExpression()) {
+    if (parentPath.isVariableDeclaration() || parentPath.isAssignmentExpression() || parentPath.isClassDeclaration()) {
       return true;
     }
   } while (parentPath !== statement);
